Demo/Demo/SwiftUIComponents/CardPaymentViews/CardApprovalResultView.swift

```diff
@@ -0,0 +1,38 @@
+import SwiftUI
+
+struct CardApprovalResultView: View {
+
+    @ObservedObject var cardPaymentViewModel: CardPaymentViewModel
+
+    var body: some View {
+        switch cardPaymentViewModel.state.approveResultResponse {
+        case .idle, .loading:
+            EmptyView()
+        case .loaded(let approveResultResponse):
+            getSuccessView(approveResultResponse: approveResultResponse)
+        case .error(let errorMessage):
+            ErrorView(errorMessage: errorMessage)
+        }
+    }
+
+    func getSuccessView(approveResultResponse: CardPaymentState.CardResult) -> some View {
```

Comment:
To me `approveResultResponse` with both `result` and `response` in the name feels confusing. Can we pick one to use and update both here and the `case .loaded` above to reflect the update name?

Reply:
It looks like we call it "approval result" below in the HStack text so maybe that's a good name?